src/app/screens/Settings.tsx

```diff
@@ -385,6 +417,60 @@ function Settings() {
           )}
         </Setting>
       </div>
+
+      <h2 className="mt-12 text-2xl font-bold dark:text-white">
+        {t("nostr.title")}
+      </h2>
+      <p className="mb-6 text-gray-500 dark:text-neutral-500 text-sm">
+        <a
+          href="https://github.com/nostr-protocol/nostr"
+          target="_blank"
+          rel="noreferrer"
+          className="underline"
+        >
+          {t("nostr.title")}
+        </a>{" "}
+        {t("nostr.hint")}
+      </p>
+      <div className="shadow bg-white sm:rounded-md sm:overflow-hidden px-6 py-2 divide-y divide-black/10 dark:divide-white/10 dark:bg-surface-02dp">
+        <Setting
+          title={t("nostr.private_key.title")}
+          subtitle={t("nostr.private_key.subtitle")}
+        >
+          {!isLoading && (
+            <>
+              <div className="w-96 flex justify-end">
+                <div className="w-96 flex-auto">
+                  <Input
+                    type="text"
```

Comment:
should we make this a type password to hide it? people will otherwise share sceenshots of it.

Reply:
Good point, thanks for raising that. I'll change it.